### DISCUSSION TITLE
Need help & suggestions for integrating sunbird content player

### DISCUSSION THREAD

**Discussion by snehal0904** [Sunbird-inQuiry/Community#45] [Q&A] [answered]
Hello Team,

We are working on one project, where we want to show a courses list as per student performance and hence we have built a custom frontend to display a list of courses and play those. 

We have successfully integrated, QUML, PDF, and video players and can open and track content in these players.
Now, we want to play h5p content, so we are trying to integrate a content player with our application. We have integrated this player but as we are using sunbird saas / Diksha as a content library (Course content is getting created here and using content get APIs we are fetching details) we are getting CORS policy errors when we are trying to play the h5p content.

I am attaching the error screenshot we are getting while playing this, please suggest possible solutions for this.
@Krishnaj20 @pallakartheekreddy @coolbung @parthlawate @itsvick 

![Screenshot from 2022-11-28 09-54-35](https://user-images.githubusercontent.com/17213411/204449047-4e77e9be-9f6b-4348-8f8d-4627f9056081.png)

**Comment by pallakartheekreddy** [comment, marked as answer]
@snehal0904 Thanks for reaching us. The inquiry building block handles only QuML Player. The remaining players belong to the Sunbird Knowlg building block. Can you please raise this issue in the Sunbird Knowlg discussion forum. Please find the link below for the same
https://github.com/Sunbird-knowlg/Community/discussions 
@vinukumar-vs @Krishnaj20

**Reply by snehal0904** [reply]
@pallakartheekreddy Thank you! Posted this on the Sunbird Knowlg discussion forum - https://github.com/Sunbird-Knowlg/Community/discussions/81

**Reply by Krishnaj20** [reply]
@snehal0904  - the inquiry & knowlg microsites clearly mention what players are supported by inQuiry & Knowlg building blocks. Previously when you have installed these players you should have noticed this in the microsite documentation. Wanted to know if there is anything incorrect you have noticed in the existing inQuiry & Knowlg microsites which needs to be corrected. 

@coolbung @parthlawate
